### ISSUE TITLE
cant get messages from a queue

### ISSUE THREAD

**Issue by ghost** [profmaad/librabbitmq-objc#5]
in AMQPConsumer change all of your

``` objc
result = amqp_simple_wait_frame(channel.connection.internalConnection, &frame);
if(result <= 0) { return nil; }
```

to

``` objc
result = amqp_simple_wait_frame(channel.connection.internalConnection, &frame);
if(result < 0) { return nil; }
```

i wasted 3 days using this wrapper thinking i was doing something wrong when it turned out the result have to be < 0 to be wrong, you have that condition wrong

**Comment by profmaad** [comment]
It worked a few years back when I wrote and used it. ;)

The problem is: I haven't used it since, since the use case vanished on me.
Any changed in librabbitmq-c since then are basically lost on this code.

After looking at the current version of rabbitmq-c, it turns out it now has to be:

``` objc
if(result != 0) { return nil; }
```

**Comment by profmaad** [comment]
Fixed in 6b3b76872c4d94cb4d815328e2f89d923ea56f44
